client/src/cmdhf15.c

```diff
@@ -222,6 +223,16 @@ static const productName_t uidmapping[] = {
     { 0, 0, "no tag-info available" } // must be the last entry
 };
 
+static inline void reverse(uint8_t *buff, uint8_t length) {
+    uint8_t upper_bound = (length % 2 == 0) ? length / 2 : (length / 2) + 1;
+    uint8_t tmp = 0;
+    for (int start = 0, end = length - 1; end >= upper_bound; ++start, --end) {
+        tmp = buff[end];
+        buff[end] = buff[start];
+        buff[start] = tmp;
+    }
+}
+
```

Comment:
I think we have a array reverse function already...   look in utils.c  or in common/